pkg/config/config_template.yaml

```diff
@@ -1057,7 +1057,7 @@ api_key:
 # compliance_config:
 
   ## @param enabled - boolean - optional - default: false
-  ## Set to true to enable continuous compliance checks
+  ## Set to true to enable the Cloud Security Posture Management (CSPM) product.
```

Comment:
```suggestion
  ## Set to true to enable Cloud Security Posture Management (CSPM).
```

Reply:
The goal of this PR was to clearly specify that is was a product and not a feature, meaning it will have a cost for the customer. What do you think about keeping the word "product" ?

Reply:
We discussed about this offline with the team, I'm going to update the PR with your suggestion. Thanks a lot